src/INITIAL_STATE.js

```diff
@@ -7,7 +7,7 @@ export default {
   ui: {
     chatHost: CHAT_HOST_STRIMS,
     chatSize: localStorage ? Number(localStorage.getItem('chatSize')) || 400 : 400,
-    showChat: true,
+    showChat: localStorage ? localStorage.getItem('showChat') === 'true' : true,
```

Comment:
Actually this would hide the chat for new users 🤔 gotta redo this

Reply:
I think:
`localStorage ? !(localStorage.getItem('showChat') === 'false') : true`
would work if you were looking for a similar option. 
Might be cleaner to change showChat to hideChat and refactor though =/. sorry about that.